usaspending_api/download/filestreaming/csv_generation.py

```diff
@@ -388,3 +392,13 @@ def fail_download(download_job, exception, message):
     download_job.error_message = '{}:\n{}'.format(message, stack_trace)
     download_job.job_status_id = JOB_STATUS_DICT['failed']
     download_job.save()
+
+
+def add_data_dictionary_to_zip(working_dir, zip_file_path):
+    write_to_log(message="Adding data dictionary to zip file")
+    data_dictionary_file_name = "Data_Dictionary_Crosswalk.xlsx"
+    data_dictionary_file_path = os.path.join(working_dir, data_dictionary_file_name)
+    # URL needs to be updated if filename changes
+    data_dictionary_url = "https://files.usaspending.gov/docs/DATA+Transparency+Crosswalk.xlsx"
```

Comment:
Future developers won't find this out until it's too late. We should look into some way to add this to the settings file, read it from a constant where the download endpoint is defined, make it an environment variable, or maybe even read it off of the endpoint contract itself.

Reply:
We sort of have something like this in the ROSETTA_DICT_S3_PATH setting, but it is used to retrieve the file from S3 in `load_rosetta.py`.

Reply:
I would be in favor of putting the two side-by-side then

Reply:
If you take this suggestion, please also remap the `s3_public_url` local variable found in load_rosetta.py to whatever solution you use, as the exact same url is established there.

Reply:
Good point. I will make the change.